src/asynchronous/transport.rs

```diff
@@ -0,0 +1,146 @@
+use std::io::{Error as IoError, Result as IoResult};
```

Comment:
How about moving this to transport dir as mod.rs? I think it is weird to have a file and a dir with the same name.

Reply:
Done